assets/svg/date-range.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This naming came from MaterialUI Icons (which I know we're not using directly) because there is a lot of different calendar icons with very specific uses, this one is specifically for a calendar with data range:

<img width="882" alt="Screenshot 2020-11-25 at 10 15 03" src="https://user-images.githubusercontent.com/119352/100214142-69215200-2f07-11eb-9bae-d8d951805e23.png">

Reply:
Oh wow, well in that case, that's what it should be! 😄  Thanks for clarifying 👍